automa_ai/agents/react_langgraph_agent.py

```diff
@@ -12,14 +11,13 @@
 
 from automa_ai.common.base_agent import BaseAgent
 from automa_ai.common.response_parser import extract_and_parse_json
+from automa_ai.common.setup_logging import setup_file_logger
```

Comment:
Can you set the same in the langgraph_chatagent? 
I dont think adding log to the chatagent would conflict with the agent here.